pyproject.toml

```diff
@@ -0,0 +1,78 @@
+[tool.poetry]
+name = "haas"
+version = "0.0.0"
+description = "Extensible Python Test Runner"
+authors = ["Simon Jagoe <simon@scalative.com>"]
+license = "BSD"
+readme = "README.rst"
+repository = "https://github.com/scalative/haas"
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: MacOS",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: OS Independent",
+    "Operating System :: POSIX",
+    "Operating System :: Unix",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Testing",
+]
+include = [
+    { path = "docs", format = "sdist" },
+    { path = "*.rst", format = "sdist" },
+    { path = "*requirements.txt", format = "sdist" },
+    { path = "tox.ini", format = "sdist" },
+]
+
+[tool.poetry.scripts]
+haas = "haas.main:main"
+
+[tool.poetry.plugins."haas.hooks.environment"]
+coverage = "haas.plugins.coverage:Coverage"
+
+[tool.poetry.plugins."haas.discovery"]
+default = "haas.plugins.discoverer:Discoverer"
+
+[tool.poetry.plugins."haas.runner"]
+default = "haas.plugins.runner:BaseTestRunner"
+parallel = "haas.plugins.parallel_runner:ParallelTestRunner"
+
+[tool.poetry.plugins."haas.result.handler"]
+default = "haas.plugins.result_handler:StandardTestResultHandler"
+quiet = "haas.plugins.result_handler:QuietTestResultHandler"
+verbose = "haas.plugins.result_handler:VerboseTestResultHandler"
+timing = "haas.plugins.result_handler:TimingResultHandler"
+
+[tool.poetry.dependencies]
+python = "^3.7"
+statistics = "^1.0.3.5"
+stevedore = [
+    { version = "^3.5.2", python = "<3.8" },
+    { version = "^4.1.1", python = ">=3.8" }
```

Comment:
Do we really need to be so strict in the runtime dependencies? Stevedore is stable enough and I think it is save to have ">=4.1.1"

Reply:
FYI, locally I build haas using stevedore 5.x

Reply:
It's typical in application development to pin dependencies to semantic-version compatible. I'm open to either. Or to adding v5 as the version used for later python versions.